src/commands/stac-github-import/stac.github.import.ts

```diff
@@ -47,6 +47,14 @@ export const commandStacGithubImport = command({
     registerCli(this, args);
 
     const gh = new GithubApi(args.repoName);
+    const validRepos = ['linz/elevation', 'linz/imagery'];
```

Comment:
I think technically would would fine, however I think it is a little difficult to read, about something like this

```typescript
const BotEmails = {
 'linz/elevation': 'elevation@linz.govt.nz'
 'linz/imagery': 'imagery@linz.govt.nz'
}

const botEmail = BotEmails[args.repoName]
if (botEmail == null) throw new Error(...)
```

My primary concern is the array split, I think its considered a unchecked array access which will start erroring when we upgrade to the next typescript, even though we know there should always be 2 records in the split you still need to go and look at all the values in the array to be sure.